### PR TITLE
Adding `inj_compr` lemma in ssrfun.

### PR DESCRIPTION
**Pull request by CohenCyril**
**Kind:** feature

I added a missing basic theorem about injectivity to `plugins/ssr/ssrfun.v`.
See math-comp/math-comp#432

- [x] Entry added in the changelog (see https://github.com/coq/coq/tree/master/doc/changelog#unreleased-changelog for details).

### CODE REVIEW

File: doc/changelog/06-ssreflect/11136-inj_compr.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+- added lemma inj_compr (`#11136 <https://github.com/coq/coq/pull/11136>`_, by Cyril Cohen).
```
Comment (Zimmi48) [inline doc/changelog/06-ssreflect/11136-inj_compr.rst]:
```suggestion
- Added lemma :g:`inj_compr` to :g:`ssr.ssrfun`
  (`#11136 <https://github.com/coq/coq/pull/11136>`_, by Cyril Cohen).
```

**Comment by Zimmi48** [comment]
Please no @-mentions in commit messages because this means getting notified every time the commit is rebased.

**Review by gares — APPROVED** [review]

**Comment by Zimmi48** [comment]
Don't merge things in the 8.11.0 milestone while the 8.11+beta1 is still open please.

**Comment by Zimmi48** [comment]
This is even more relevant for this PR which is a feature: while it might be considered trivial enough to get accepted in the beta, it would clearly be out of scope between the beta and the final release.

**Comment by ppedrot** [comment]
Lucky you all, I'm adding this to the beta, for it is trivial indeed.

**Pull request merged**